src/components/common/CopyButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconButton } from '@rmwc/icon-button';
+import React from 'react';
+
+interface CopyButtonProps {
+  text: string;
+  label?: string;
+}
+
+// TODO(davideast): Depend on pkg https://github.com/jaredLunde/react-hook/tree/master/packages/copy#readme
+function useCopy(text: string) {
+  const [copied, setCopied] = React.useState(false);
```

Comment:
Looks like we import useState/useRef directly everywhere else in the app, e.g. `useState(false);`